packages/storage/src/common/urls.ts

```diff
@@ -0,0 +1,45 @@
+import { GatewayUrls } from "../types";
+
+/**
+ * @internal
+ */
+export const DEFAULT_GATEWAY_URLS: GatewayUrls = {
+  // Note: Gateway URLs should have trailing slashes (we clean this on user input)
+  "ipfs://": [
+    "https://gateway.ipfscdn.io/ipfs/",
+    "https://cloudflare-ipfs.com/ipfs/",
+    "https://ipfs.io/ipfs/",
```

Comment:
always found it error prone that you have to pass the `/ipfs/` at the end. so easy to forget

Reply:
I'm thinking of this as a generic string match/replace - issue is that for other schemes, there's not necessarily an obvious thing to add on top like that - and would like the replacement to be as explicit as possible for the user. 

Definitely might be nice to do some additional checks for the user though